### DISCUSSION TITLE
Posibility to limit access to a certain site in admin for multisite

### DISCUSSION THREAD

**Discussion by niekappartmedia** [statamic/cms#4878] [Q&A]
Dear statamic users,

Currently we are investigating an option to limit the acces for a user to content of a certain site in a multi site installation.
We didn't found anything yet that comes close to what we want.
We are now thinking to create it by our self but is it possible to limit the content and site action?

Mayby by manipulating the config or creating seperate page collections.

Do you guys have a good idea how we could accomplish this wish?

Kind regards,

Niek

**Comment by jackmcdade** [comment]
We have an open issue for that as it's not a feature that currently exists: https://github.com/statamic/cms/issues/2667
